docs/pages/try-out-teleport/local-kubernetes.mdx

```diff
@@ -125,8 +125,9 @@ HTTP API is working:
 
 ```code
 $ EXTERNAL_IP=$(kubectl get service teleport-cluster -o jsonpath='{ .status.loadBalancer.ingress[0].ip }')
-$ curl --insecure https://${EXTERNAL_IP?}:443/webapi/ping
-{"auth":{"type":"local","second_factor":"otp","preferred_local_mfa":"otp","has_motd":false},"proxy":{"kube":{"enabled":true,"listen_addr":"0.0.0.0:3026"},"ssh":{"listen_addr":"[::]:3023","tunnel_listen_addr":"0.0.0.0:3024","public_addr":"teleport-cluster:443"},"db":{"mysql_listen_addr":"0.0.0.0:3036"},"tls_routing_enabled":false},"server_version":"8.2.0","min_client_version":"7.0.0"}%
+$ TELEPORT_PING_URL="https://${EXTERNAL_IP?}:443/webapi/ping"
```

Comment:
Could this and the line above be combined? If we're not going to use the `EXTERNAL_IP` var except to create the `TELEPORT_PING_URL` var, why not add the logic from the former into the latter? I don't have a working cluster to test this, but something like:

```suggestion
$ TELEPORT_PING_URL="https://$(kubectl get service teleport-cluster -o jsonpath='{ .status.loadBalancer.ingress[0].ip }'):443/webapi/ping"
```

Reply:
Never mind, I see we use `EXTERNAL_IP` later on. I'll add a comment here to explain.